### DISCUSSION TITLE
translation doesnt work in script tag !!!

### DISCUSSION THREAD

**Discussion by guardesign** [nuxt-modules/i18n#2322] [Q&A]
### Environment

HI 
when I want to 118 inside the Nuxt-3 project to translate strings inside the script or in app-config.js it doesn't work anymore ! and that is awful !!! could you please help me how can handle this issue !!

### Reproduction

when I want to 118 inside the Nuxt-3 project to translate strings inside the script or in app-config.js it doesn't work anymore ! and that is awful !!! could you please help me how can handle this issue !!

### Describe the bug

when I want to 118 inside the Nuxt-3 project to translate strings inside the script or in app-config.js it doesn't work anymore ! and that is awful !!! could you please help me how can handle this issue !!

**Comment by kazupon** [comment]
This is a question, not an issue. we will move to github disscussion.

**Reply by kazupon** [reply]
> when I want to 118 inside the Nuxt-3 project to translate strings inside the script or in app-config.js it doesn't work anymore ! and that is awful !!! could you please help me how can handle this issue !!

We need more examples to help you. Can you give it to us please? 😅
